src/server/terrain/biomes.zig

```diff
@@ -760,6 +764,22 @@ pub fn getById(id: []const u8) *const Biome {
 	};
 }
 
+pub fn getByNumericId(index: u32, defaultIndex: ?u32) !*const Biome {
```

Comment:
```suggestion
pub fn getByIndex(index: u32, defaultIndex: ?u32) !*const Biome {
```

Reply:
Also, if the second parameters isn't used, then maybe not add the extra complexity it brings for now?

Reply:
Actually, considering how easy to use is zigs `orelse` we should never have this argument.